kornia/augmentation/auto/operations/base.py

```diff
@@ -143,7 +143,7 @@ def forward(self, input: Tensor, params: Optional[Dict[str, Tensor]] = None) ->
         if params is None:
             params = self.forward_parameters(input.shape)
 
-        batch_prob = params["batch_prob"][(...,) + ((None,) * (len(input.shape) - 1))]
+        batch_prob = params["batch_prob"][(...,) + ((None,) * (len(input.shape) - 1))].to(device=input.device)
```

Comment:
Not sure if this is the right place to move the `batch_prob` to the right device (@shijianjian)